payments/price_feed.py

```diff
@@ -1,16 +1,27 @@
 import requests
-
 import config
 
 
 def get_price(currency):
-    price_feed = "https://api.coindesk.com/v1/bpi/currentprice.json"
+
+    # Define some currency_provider-specific settings 
+    if config.currency_provider == "COINDESK":
```

Comment:
It would be nice to make `COINDESK` a constant. It is a bit magical having a string

Reply:
Fine hardcoded like this I think. It's what i've done everywhere else 😅

Maybe if we add a third currency provider we can start to separate out into functions.

Reply:
@jkar32  Externalized the constants into a dict, in a new function: https://github.com/ingolevin/SatSale/commit/3b206727eb4427d59c15a45d9ffafecc0eaab963

Reply:
@ingolevin Not so much a dict. Normally with constant in python, we have a constants file and then all the constants are upperface. E.g.
```
> constants.py
COINDESK = "COINDESK"
SECOND_THING = "SECOND_THING"
```